services/elasticache/pkg/resource/replication_group/custom_set_output.go

```diff
@@ -55,11 +55,4 @@ func (rm *resourceManager) customSetOutput(
 	ko *svcapitypes.ReplicationGroup,
 ) {
 	// TODO: custom code
-	if ko.Status.NodeGroups != nil {
-		for _, nodegroup := range ko.Status.NodeGroups {
-			membersCount := int64(len(nodegroup.NodeGroupMembers))
-			ko.Spec.ReplicasPerNodeGroup = &membersCount
-			break
-		}
-	}
```

Comment:
++ I presume you removed this because it's not appropriate to have controller code modifying the `Spec` attributes (only the `Status` attributes)

Reply:
yes, as both `desired` and `latest` resources are available in update method call, there was no need to update ` Spec`  on latest.